Rework bootstrapping docs (#2129)
chef/chef-web-docs: chef_master/source/run_lists.rst

.. tag chef_client_bootstrap_node

> A node is any physical, virtual, or cloud system that is configured to be maintained by a Chef Infra Client. Bootstrapping installs Chef Infra Client on a target system so that it can run as a client and sets the node up to communicate with a Chef Infra Server. There are two ways to do this:

Reviewer: ```suggestion
A node is any physical, virtual, or cloud device that is configured and maintained by an instance of Chef Infra Client. Bootstrapping installs Chef Infra Client on a target system so that it can run as a client and sets the node up to communicate with a Chef Infra Server. There are two ways to do this:
```

* Run the ``knife bootstrap`` command from a workstation.

> * Perform an unattended install to bootstrap from the node itself, without requiring SSH  or WinRM connectivity

Reviewer: ```suggestion
* Perform an unattended install to bootstrap from the node itself, without requiring SSH or WinRM connectivity.
```

   * - Stages
     - Description
   * - **$ knife bootstrap**

>      - The ``knife bootstrap`` subcommand is issued from a workstation. The hostname, IP address, or FQDN of the target node is issued as part of this command. An SSH connection is established with the target node using port 22. A shell script is assembled using the chef-full.erb (the default bootstrap template), and is then executed on the target node.

Reviewer: ```suggestion
     - Enter the ``knife bootstrap`` subcommand from a workstation. Include the hostname, IP address, or FQDN of the target node as part of this command. Knife will establish an SSH connection with the target node using port 22. Knife assembles a shell script using the chef-full.erb file, which is the default bootstrap template.
```
